# Detected Target
std_msgs/Header header
string  id
float64 x
float64 y
float64 z
float64 var_x
float64 var_y
float64 var_z
